src/main/groovy/life/qbic/datamodel/dtos/projectmanagement/Project.groovy

```diff
@@ -0,0 +1,35 @@
+package life.qbic.datamodel.dtos.projectmanagement
+
+import life.qbic.datamodel.dtos.business.OfferId
+
+/**
+ * A typical scientific QBiC project.
+ *
+ * Describes and provides information about a scientific project
+ * at QBiC.
+ *
+ * @since 1.0.0
+ */
+class Project {
+
+    /**
+     * A short but descriptive project title
+     */
+    final String projectTitle
```

Comment:
Do we need a project title? How is it different from the title of the project title of the offer?

Reply:
it is not, but it is a context change we have here. we left the offer realm and are in the project life cycle now. I don't want to pollute contexts by mixing up objects, just to provide the information. I want it very declarative.